monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -211,6 +217,11 @@ object Rx {
       val c2 = run(other)(effect)
       Cancelable { () => c1.cancel; c2.cancel }
 
+    case MergeN(self) => {
+      val ccs: List[Cancelable] = self.map(rx => run(rx)(effect))
```

Comment:
Should we use `List` or something further up the inheritance hierarchy (`Iterable` ?)

Reply:
I guess if you go for varargs this is going to change to `Seq` (either way works for me)

Reply:
I'm hesitant to explicitly require `Seq`, it seems particularly bad:

https://danielasfregola.com/2015/06/15/which-immutable-scala-collection/

https://youtu.be/DXFUn90hudk?t=35m30s

Maybe we shouldn't fret too much about performance at this point, bench marks in use cases can be done later, as we aren't at 1.0 yet.